### PR TITLE
Updated Newtonsoft.Json to 13.0.1 to address reported vulnerability

### PR DESCRIPTION
**Pull request by lordkyzr**
Version bump to 4.1.1

### CODE REVIEW

File: src/iovation.LaunchKey.Sdk.Tests/iovation.LaunchKey.Sdk.Tests.csproj [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <TargetFrameworks>netcoreapp21;net45</TargetFrameworks>
 
     <IsPackable>false</IsPackable>
-    <ReleaseVersion>4.1.0</ReleaseVersion>
+    <ReleaseVersion>4.1.10</ReleaseVersion>
```
[the remark this answers is not in the document]
Comment (lordkyzr) [reply src/iovation.LaunchKey.Sdk.Tests/iovation.LaunchKey.Sdk.Tests.csproj]: This should have been .1 not .10. Good catch!

File: src/iovation.LaunchKey.Sdk/iovation.LaunchKey.Sdk.csproj [hunk 2]
```diff
@@ -1,8 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>netstandard20;net45</TargetFrameworks>
-    <AssemblyVersion>4.1.0.0</AssemblyVersion>
-    <FileVersion>4.1.0.0</FileVersion>
+    <AssemblyVersion>4.1.1.0</AssemblyVersion>
+    <FileVersion>4.1.1.0</FileVersion>
     <Version>4.1.0.0</Version>
```
[the remark this answers is not in the document]
Comment (lordkyzr) [reply src/iovation.LaunchKey.Sdk/iovation.LaunchKey.Sdk.csproj]: Fixed

File: src/iovation.LaunchKey.Sdk/iovation.LaunchKey.Sdk.csproj [hunk 3]
```diff
@@ -13,13 +13,14 @@
     <PackageProjectUrl>https://docs.launchkey.com/api/index.html</PackageProjectUrl>
     <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
     <ReleaseVersion>4.1.0</ReleaseVersion>
```
[the remark this answers is not in the document]
Comment (lordkyzr) [reply src/iovation.LaunchKey.Sdk/iovation.LaunchKey.Sdk.csproj]: It should! Fixed!

**Review by CMoncur — APPROVED** [review]
:shipit:

**Review by bporter125 — APPROVED** [review]

**Pull request merged**